include/vrv/svgdevicecontext.h

```diff
@@ -220,7 +221,12 @@ class SvgDeviceContext : public DeviceContext {
      */
     void SetAdditionalAttributes(std::vector<std::string> additionalAttributes)
     {
-        for (std::string s : additionalAttributes) m_svgAdditionalAttributes.push_back(s);
+        for (std::string s : additionalAttributes) {
+            std::string className = s.substr(0, s.find("@")); // parse <element@attribute>, e.g., "note@pname"
+            std::string attributeName = s.substr(s.find("@") + 1);
+            ClassId classId = ObjectFactory::GetInstance()->GetClassId(className);
+            m_svgAdditionalAttributes.insert({ classId, attributeName });
+        }
```

Comment:
Do we have to check whether the classId (or the attribute string) exists before inserting it to the multimap?